examples/CMakeLists.txt

```diff
@@ -42,130 +42,37 @@ project(examples)
 # cmake needs is the dlib source code folder and it will take care of everything.
 add_subdirectory(../dlib dlib_build)
 
-# If you have cmake 3.14 or newer you can even use FetchContent instead of
```

Comment:
Keep the tutoral text (with some adjustments to remove the comments about old cmake stuff).  People totally need the tutorial, I've received no end of questions about how to make a CMakeLists.txt file and this tutorial has been hugely helpful for that.

Reply:
Do you really want to be in the game of teaching people how to use cmake? If they can't go to the cmake website and read the docs, then they can't be helped...

Reply:
People won't let me not be in that game

Reply:
Okidok

Reply:
There isn't an example that uses (and checks the functionality of) `find_package(dlib)`.
There should probably be a unit test for that. Basically a check for the install functionality.